hw/sensor/creator/src/sensor_creator.c

```diff
@@ -1126,6 +1159,54 @@ config_bma2xx_sensor(void)
 }
 #endif
 
+#if MYNEWT_VAL(BMP388_OFB)
+static int
+config_bmp388_sensor(void)
+{
+    int rc;
+    struct os_dev *dev;
+    struct bmp388_cfg cfg = {0};
+
+    dev = (struct os_dev *) os_dev_open("bmp388_0", OS_TIMEOUT_NEVER, NULL);
+    assert(dev != NULL);
+
+	cfg.rate = BMP3_ODR_50_HZ;
+
+    /*options: BMP388_DRDY_INT, BMP388_FIFO_WTMK_INT, BMP388_FIFO_FULL_INT */
+	cfg.int_enable_type = BMP388_FIFO_FULL_INT;
+
+    cfg.int_pp_od = 0;
+    cfg.int_latched = 0;
+    cfg.int_active_low = 1;
+
+
+    /* options: BMP388_FIFO_M_BYPASS, BMP388_FIFO_M_FIFO */
+    cfg.fifo_mode = BMP388_FIFO_M_FIFO;
+	cfg.fifo_threshold = 73;
+
+	cfg.filter_press_osr = BMP3_OVERSAMPLING_2X;
+	cfg.filter_temp_osr = BMP3_OVERSAMPLING_2X;
+    cfg.power_mode = BMP3_FORCED_MODE;
+
+    /* options: BMP388_READ_M_POLL or BMP388_READ_M_STREAM */
+	cfg.read_mode.mode = BMP388_READ_M_STREAM;
+
+    /* options: BMP388_DRDY_INT,  BMP388_FIFO_WTMK_INT, BMP388_FIFO_FULL_INT */
+	cfg.read_mode.int_type = BMP388_FIFO_FULL_INT;
+	cfg.read_mode.int_num = MYNEWT_VAL(BMP388_INT_NUM);
+    cfg.mask = SENSOR_TYPE_AMBIENT_TEMPERATURE|
+                       SENSOR_TYPE_PRESSURE;
+

```

Comment:
Indentation seems messed up in this function. Can you please fix it. We use 4 spaces.